sdk/go/pulumi/context_test.go

```diff
@@ -625,3 +626,37 @@ func TestWithValue(t *testing.T) {
 	assert.Equal(t, val, newCtx.Value(key))
 	assert.Equal(t, newCtx.state, testCtx.state)
 }
+
+func TestInvokeOutput(t *testing.T) {
```

Comment:
Note that this test does not fail without the code change above. Output guards against a fulfilling, but we shouldn't rely on this behaviour.